imports/ui/views/users/UserPage/UserContainer.jsx

```diff
@@ -1,13 +1,41 @@
 import { Meteor } from 'meteor/meteor'
 import { createContainer } from 'meteor/react-meteor-data'
-import React, { Component } from 'react'
+import React, { Component, PropTypes } from 'react'
 
+import { Experiences } from '/imports/api/Experiences'
 import { Users } from '/imports/api/Users'
+import { Tags } from '/imports/api/Tags'
 import UserPage from './UserPage'
 
-export default createContainer((props) => {
-    const query = {_id: props.params.id}
-    const handle = Meteor.subscribe('Users.get', query)
-    if (!handle.ready()) return { loading: true, user: {} }
-    else return { loading: false, user: Users.findOne(query) }
+const UserContainer = createContainer((props) => {
+  // get organization
+  const query = {_id: props.params.id}
+  const userHandle = Meteor.subscribe('Users.get', query)
+  if (!userHandle.ready()) return { loading: true, user: {} }
+  let user = Users.findOne(query)
+
+  // get tags
```

Comment:
You could maybe put the the subscriptions for getting the tags and experiences into an array since those two subscriptions happen independently of one another with something like this

const subs = [
    Meteor.subscribe('Experiences.get', experiencesQuery),
    Meteor.subscribe('Tags.get', tagQuery)
  ]

  if (_.some(subs, (s) => !s.ready())) return { loading: true, user: {} }
 // update organization with tag and experience objects